apps/files/src/store.js

```diff
@@ -1,12 +1,29 @@
-import { findIndex, without } from 'lodash'
-
+import { findIndex, without, map, filter } from 'lodash'
 const namespaced = true
 
 const state = {
   currentFolder: null,
   files: [],
+  filesSearched: [],
+  fileFilter: [
+    {
+      name: 'Files',
```

Comment:
v-translate missing - reasonable within the store?

Reply:
I know it was missing before. I'm questioning the move of frontend strings into the store ;-)

Reply:
You're right :( I need to add them.

Reply:
still missing -> new PR please @tempelgogo